README-UBUNTU-14.04

```diff
@@ -0,0 +1,14 @@
+Setup steps - non hdfs
+1)cd scribe; rm -f debian; ln -sf debian_nonhdfs debian
+2)sudo vi /etc/apt/sources.list.d/inmobi.list
+add the entry deb http://repo.corp.inmobi.com:9999/precise ./
+3)sudo apt-get update
+4)sudo apt-get install debhelper autotools-dev inmobi-techplatform-libthrift-dev inmobi-techplatform-libfb303 libboost-thread-dev  libstdc++6-4.4-dev libevent-dev inmobi-techplatform-thrift-compiler libboost-dev libboost-test-dev libboost-program-options-dev libboost-system-dev libboost-filesystem-dev libevent-dev automake libtool flex bison pkg-config g++ libssl-dev
+5) creating sym links for pointing the multi threaded libboost to single threaded libboost
```

Comment:
https://groups.google.com/forum/#!topic/boost-list/5nbpE0J_E1c - dicussion on the same. 
http://permalink.gmane.org/gmane.comp.lib.boost.user/62273
libboost-thread-mt and libboost-thread were essentially same and the "-mt" part was a decorator. And now there is no more "-mt" version available in the latest versions of libboost. http://www.boost.org/users/history/version_1_40_0.html
From the build package session. 

"The default naming of libraries in Unix-like environment now matches system conventions, and does not include various decorations"